tests/integration/hpe_3par_manager.py

```diff
@@ -180,6 +183,64 @@ def hpe_inspect_volume(self, volume, **kwargs):
 
         return inspect_volume
 
+    def hpe_inspect_share(self, volume, **kwargs):
+        inspect_volume = self.client.inspect_volume(volume['Name'])
+        self.assertEqual(volume['Name'], inspect_volume['Name'])
+        self.assertEqual(volume['Driver'], inspect_volume['Driver'])
+        self.assertEqual(volume['Options'], inspect_volume['Options'])
+        self.assertIn('Status', inspect_volume)
+        timeout = time.time() + 600
+        while 1:

```

Comment:
instead while 1, we can have have while around the status and timeout

Reply:
The wsapi on retrieving tasks does not have any reference to either share name, fpg or vfs on which I could query to get the specific task id. Therefore retaining the timeout 

output of task
RESP BODY:{"total":145,"members":[{"id":6867,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 00:15:00 IST","finishTime":"2019-06-03 00:15:00 IST","user":"3parsvc"},{"id":6868,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 00:27:00 IST","finishTime":"2019-06-03 00:27:00 IST","user":"3parsvc"},{"id":6869,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 01:15:00 IST","finishTime":"2019-06-03 01:15:00 IST","user":"3parsvc"},{"id":6870,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 01:27:00 IST","finishTime":"2019-06-03 01:27:00 IST","user":"3parsvc"},{"id":6871,"type":14,"name":"schedule1","status":1,"startTime":"2019-06-03 02:10:00 IST","finishTime":"2019-06-03 02:10:00 IST","user":"3paradm"},{"id":6872,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 02:15:00 IST","finishTime":"2019-06-03 02:15:00 IST","user":"3parsvc"},{"id":6873,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 02:27:00 IST","finishTime":"2019-06-03 02:27:00 IST","user":"3parsvc"},{"id":6874,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 03:15:00 IST","finishTime":"2019-06-03 03:15:00 IST","user":"3parsvc"},{"id":6875,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 03:27:00 IST","finishTime":"2019-06-03 03:27:00 IST","user":"3parsvc"},{"id":6876,"type":15,"name":"sample","status":1,"startTime":"2019-06-03 03:37:00 IST","finishTime":"2019-06-03 03:55:25 IST","user":"3parsvc"},{"id":6877,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 04:15:00 IST","finishTime":"2019-06-03 04:15:00 IST","user":"3parsvc"},{"id":6878,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 04:27:00 IST","finishTime":"2019-06-03 04:27:00 IST","user":"3parsvc"},{"id":6879,"type":15,"name":"coremanagement","status":1,"startTime":"2019-06-03 04:47:00 IST","finishTime":"2019-06-03 04:47:00 IST","user":"3parsvc"},{"id":6880,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 05:15:00 IST","finishTime":"2019-06-03 05:15:00 IST","user":"3parsvc"},{"id":6881,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 05:27:00 IST","finishTime":"2019-06-03 05:27:00 IST","user":"3parsvc"},{"id":6882,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 06:15:00 IST","finishTime":"2019-06-03 06:15:00 IST","user":"3parsvc"},{"id":6883,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 06:27:00 IST","finishTime":"2019-06-03 06:27:00 IST","user":"3parsvc"},{"id":6884,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 07:15:00 IST","finishTime":"2019-06-03 07:15:00 IST","user":"3parsvc"},{"id":6885,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 07:27:00 IST","finishTime":"2019-06-03 07:27:01 IST","user":"3parsvc"},{"id":6886,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 08:15:00 IST","finishTime":"2019-06-03 08:15:00 IST","user":"3parsvc"},{"id":6887,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 08:27:00 IST","finishTime":"2019-06-03 08:27:01 IST","user":"3parsvc"},{"id":6888,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 09:15:00 IST","finishTime":"2019-06-03 09:15:00 IST","user":"3parsvc"},{"id":6889,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 09:27:00 IST","finishTime":"2019-06-03 09:27:01 IST","user":"3parsvc"},{"id":6890,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 10:15:00 IST","finishTime":"2019-06-03 10:15:00 IST","user":"3parsvc"},{"id":6891,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 10:27:00 IST","finishTime":"2019-06-03 10:27:01 IST","user":"3parsvc"},{"id":6892,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 10:53:14 IST","finishTime":"2019-06-03 10:53:54 IST","user":"3paradm"},{"id":6893,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 11:15:00 IST","finishTime":"2019-06-03 11:15:00 IST","user":"3parsvc"},{"id":6894,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 11:27:00 IST","finishTime":"2019-06-03 11:27:01 IST","user":"3parsvc"},{"id":6895,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 11:41:29 IST","finishTime":"2019-06-03 11:42:58 IST","user":"3paradm"},{"id":6896,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 11:43:04 IST","finishTime":"2019-06-03 11:43:29 IST","user":"3paradm"},{"id":6897,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:08:00 IST","finishTime":"2019-06-03 12:08:40 IST","user":"3paradm"},{"id":6898,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:11:14 IST","finishTime":"2019-06-03 12:11:54 IST","user":"3paradm"},{"id":6899,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:13:30 IST","finishTime":"2019-06-03 12:14:41 IST","user":"3paradm"},{"id":6900,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 12:13:43 IST","finishTime":"2019-06-03 12:15:02 IST","user":"3paradm"},{"id":6901,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 12:14:04 IST","finishTime":"2019-06-03 12:16:24 IST","user":"3paradm"},{"id":6902,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 12:15:00 IST","finishTime":"2019-06-03 12:15:00 IST","user":"3parsvc"},{"id":6903,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 12:15:05 IST","finishTime":"2019-06-03 12:15:30 IST","user":"3paradm"},{"id":6904,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 12:16:29 IST","finishTime":"2019-06-03 12:17:04 IST","user":"3paradm"},{"id":6905,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 12:19:58 IST","finishTime":"2019-06-03 12:21:07 IST","user":"3paradm"},{"id":6906,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 12:21:13 IST","finishTime":"2019-06-03 12:21:38 IST","user":"3paradm"},{"id":6907,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:21:39 IST","finishTime":"2019-06-03 12:22:39 IST","user":"3paradm"},{"id":6908,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 12:27:00 IST","finishTime":"2019-06-03 12:27:01 IST","user":"3parsvc"},{"id":6909,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:27:07 IST","finishTime":"2019-06-03 12:27:47 IST","user":"3paradm"},{"id":6910,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 12:27:40 IST","finishTime":"2019-06-03 12:29:32 IST","user":"3paradm"},{"id":6911,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 12:29:43 IST","finishTime":"2019-06-03 12:30:18 IST","user":"3paradm"},{"id":6912,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 12:39:08 IST","finishTime":"2019-06-03 12:40:22 IST","user":"3paradm"},{"id":6913,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 12:40:31 IST","finishTime":"2019-06-03 12:40:55 IST","user":"3paradm"},{"id":6914,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 12:46:56 IST","finishTime":"2019-06-03 12:47:36 IST","user":"3paradm"},{"id":6915,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 13:15:00 IST","finishTime":"2019-06-03 13:15:00 IST","user":"3parsvc"},{"id":6916,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 13:27:00 IST","finishTime":"2019-06-03 13:27:01 IST","user":"3parsvc"},{"id":6917,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 14:03:30 IST","finishTime":"2019-06-03 14:04:15 IST","user":"3paradm"},{"id":6918,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 14:15:00 IST","finishTime":"2019-06-03 14:15:00 IST","user":"3parsvc"},{"id":6919,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 14:20:33 IST","finishTime":"2019-06-03 14:21:18 IST","user":"3paradm"},{"id":6920,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 14:21:34 IST","finishTime":"2019-06-03 14:22:49 IST","user":"3paradm"},{"id":6921,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 14:22:57 IST","finishTime":"2019-06-03 14:23:32 IST","user":"3paradm"},{"id":6922,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 14:27:00 IST","finishTime":"2019-06-03 14:27:01 IST","user":"3parsvc"},{"id":6923,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 14:38:59 IST","finishTime":"2019-06-03 14:39:49 IST","user":"3paradm"},{"id":6924,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 14:41:43 IST","finishTime":"2019-06-03 14:42:57 IST","user":"3paradm"},{"id":6925,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 14:43:05 IST","finishTime":"2019-06-03 14:43:40 IST","user":"3paradm"},{"id":6926,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 14:56:24 IST","finishTime":"2019-06-03 14:57:14 IST","user":"3paradm"},{"id":6927,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 14:59:56 IST","finishTime":"2019-06-03 15:01:10 IST","user":"3paradm"},{"id":6928,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 15:01:18 IST","finishTime":"2019-06-03 15:01:53 IST","user":"3paradm"},{"id":6929,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 15:07:08 IST","finishTime":"2019-06-03 15:07:54 IST","user":"3paradm"},{"id":6930,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 15:15:00 IST","finishTime":"2019-06-03 15:15:00 IST","user":"3parsvc"},{"id":6931,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 15:20:25 IST","finishTime":"2019-06-03 15:21:39 IST","user":"3paradm"},{"id":6932,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 15:21:48 IST","finishTime":"2019-06-03 15:22:23 IST","user":"3paradm"},{"id":6933,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 15:26:32 IST","finishTime":"2019-06-03 15:27:51 IST","user":"3paradm"},{"id":6934,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 15:27:00 IST","finishTime":"2019-06-03 15:27:01 IST","user":"3parsvc"},{"id":6935,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 15:27:57 IST","finishTime":"2019-06-03 15:28:32 IST","user":"3paradm"},{"id":6936,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 15:29:52 IST","finishTime":"2019-06-03 15:30:37 IST","user":"3paradm"},{"id":6937,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 15:34:36 IST","finishTime":"2019-06-03 15:35:50 IST","user":"3paradm"},{"id":6938,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 15:35:59 IST","finishTime":"2019-06-03 15:36:34 IST","user":"3paradm"},{"id":6939,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 15:40:59 IST","finishTime":"2019-06-03 15:41:45 IST","user":"3paradm"},{"id":6940,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 15:51:46 IST","finishTime":"2019-06-03 15:53:00 IST","user":"3paradm"},{"id":6941,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 15:53:11 IST","finishTime":"2019-06-03 15:53:46 IST","user":"3paradm"},{"id":6942,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 16:12:51 IST","finishTime":"2019-06-03 16:13:26 IST","user":"3paradm"},{"id":6943,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 16:13:11 IST","finishTime":"2019-06-03 16:14:21 IST","user":"3paradm"},{"id":6944,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 16:14:06 IST","finishTime":"2019-06-03 16:14:56 IST","user":"3paradm"},{"id":6945,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 16:14:27 IST","finishTime":"2019-06-03 16:14:57 IST","user":"3paradm"},{"id":6946,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 16:15:00 IST","finishTime":"2019-06-03 16:15:01 IST","user":"3parsvc"},{"id":6947,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 16:27:00 IST","finishTime":"2019-06-03 16:27:01 IST","user":"3parsvc"},{"id":6948,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 16:29:43 IST","finishTime":"2019-06-03 16:30:23 IST","user":"3paradm"},{"id":6949,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 16:31:05 IST","finishTime":"2019-06-03 16:32:19 IST","user":"3paradm"},{"id":6950,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 16:32:30 IST","finishTime":"2019-06-03 16:32:50 IST","user":"3paradm"},{"id":6951,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 16:37:37 IST","finishTime":"2019-06-03 16:38:17 IST","user":"3paradm"},{"id":6952,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 16:41:29 IST","finishTime":"2019-06-03 16:42:43 IST","user":"3paradm"},{"id":6953,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 16:42:52 IST","finishTime":"2019-06-03 16:43:16 IST","user":"3paradm"},{"id":6954,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 17:15:00 IST","finishTime":"2019-06-03 17:15:00 IST","user":"3parsvc"},{"id":6955,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:18:35 IST","finishTime":"2019-06-03 17:19:44 IST","user":"3paradm"},{"id":6956,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:19:50 IST","finishTime":"2019-06-03 17:20:25 IST","user":"3paradm"},{"id":6957,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 17:20:03 IST","finishTime":"2019-06-03 17:20:44 IST","user":"3paradm"},{"id":6958,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 17:23:34 IST","finishTime":"2019-06-03 17:24:25 IST","user":"3paradm"},{"id":6959,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 17:27:00 IST","finishTime":"2019-06-03 17:27:00 IST","user":"3parsvc"},{"id":6960,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:29:35 IST","finishTime":"2019-06-03 17:31:04 IST","user":"3paradm"},{"id":6961,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:31:09 IST","finishTime":"2019-06-03 17:31:29 IST","user":"3paradm"},{"id":6962,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 17:34:10 IST","finishTime":"2019-06-03 17:34:50 IST","user":"3paradm"},{"id":6963,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:43:20 IST","finishTime":"2019-06-03 17:44:29 IST","user":"3paradm"},{"id":6964,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:44:35 IST","finishTime":"2019-06-03 17:44:55 IST","user":"3paradm"},{"id":6965,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 17:45:00 IST","finishTime":"2019-06-03 17:45:40 IST","user":"3paradm"},{"id":6966,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:48:10 IST","finishTime":"2019-06-03 17:49:24 IST","user":"3paradm"},{"id":6967,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:49:33 IST","finishTime":"2019-06-03 17:49:52 IST","user":"3paradm"},{"id":6968,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 17:54:51 IST","finishTime":"2019-06-03 17:55:27 IST","user":"3paradm"},{"id":6969,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:55:13 IST","finishTime":"2019-06-03 17:56:32 IST","user":"3paradm"},{"id":6970,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:56:37 IST","finishTime":"2019-06-03 17:57:02 IST","user":"3paradm"},{"id":6971,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 17:58:05 IST","finishTime":"2019-06-03 17:59:19 IST","user":"3paradm"},{"id":6972,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 17:59:30 IST","finishTime":"2019-06-03 17:59:55 IST","user":"3paradm"},{"id":6973,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 18:00:01 IST","finishTime":"2019-06-03 18:00:41 IST","user":"3paradm"},{"id":6974,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 18:15:00 IST","finishTime":"2019-06-03 18:15:00 IST","user":"3parsvc"},{"id":6975,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 18:27:00 IST","finishTime":"2019-06-03 18:27:00 IST","user":"3parsvc"},{"id":6976,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 19:15:00 IST","finishTime":"2019-06-03 19:15:00 IST","user":"3parsvc"},{"id":6977,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 19:27:00 IST","finishTime":"2019-06-03 19:27:00 IST","user":"3parsvc"},{"id":6978,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 20:15:00 IST","finishTime":"2019-06-03 20:15:00 IST","user":"3parsvc"},{"id":6979,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 20:27:00 IST","finishTime":"2019-06-03 20:27:00 IST","user":"3parsvc"},{"id":6980,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 21:15:00 IST","finishTime":"2019-06-03 21:15:00 IST","user":"3parsvc"},{"id":6981,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:26:40 IST","finishTime":"2019-06-03 21:27:30 IST","user":"3paradm"},{"id":6982,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 21:27:00 IST","finishTime":"2019-06-03 21:27:00 IST","user":"3parsvc"},{"id":6983,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 21:37:08 IST","finishTime":"2019-06-03 21:38:27 IST","user":"3paradm"},{"id":6984,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 21:38:32 IST","finishTime":"2019-06-03 21:38:57 IST","user":"3paradm"},{"id":6985,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:39:04 IST","finishTime":"2019-06-03 21:39:54 IST","user":"3paradm"},{"id":6986,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 21:41:18 IST","finishTime":"2019-06-03 21:42:36 IST","user":"3paradm"},{"id":6987,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:46:39 IST","finishTime":"2019-06-03 21:47:13 IST","user":"3paradm"},{"id":6988,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:46:52 IST","finishTime":"2019-06-03 21:47:32 IST","user":"3paradm"},{"id":6989,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:47:14 IST","finishTime":"2019-06-03 21:48:04 IST","user":"3paradm"},{"id":6990,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:47:26 IST","finishTime":"2019-06-03 21:50:02 IST","user":"3paradm"},{"id":6991,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:47:31 IST","finishTime":"2019-06-03 21:48:16 IST","user":"3paradm"},{"id":6992,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 21:48:27 IST","finishTime":"2019-06-03 21:49:41 IST","user":"3paradm"},{"id":6993,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 21:49:51 IST","finishTime":"2019-06-03 21:50:26 IST","user":"3paradm"},{"id":6994,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 21:52:05 IST","finishTime":"2019-06-03 21:53:25 IST","user":"3paradm"},{"id":6995,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 21:52:39 IST","finishTime":"2019-06-03 21:53:58 IST","user":"3paradm"},{"id":6996,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 21:53:29 IST","finishTime":"2019-06-03 21:54:04 IST","user":"3paradm"},{"id":6997,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 22:00:31 IST","finishTime":"2019-06-03 22:01:16 IST","user":"3paradm"},{"id":6998,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 22:15:00 IST","finishTime":"2019-06-03 22:15:00 IST","user":"3parsvc"},{"id":6999,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 22:19:32 IST","finishTime":"2019-06-03 22:20:07 IST","user":"3paradm"},{"id":7000,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 22:24:08 IST","finishTime":"2019-06-03 22:25:22 IST","user":"3paradm"},{"id":7001,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 22:25:32 IST","finishTime":"2019-06-03 22:26:07 IST","user":"3paradm"},{"id":7002,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 22:27:00 IST","finishTime":"2019-06-03 22:27:00 IST","user":"3parsvc"},{"id":7003,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 22:30:16 IST","finishTime":"2019-06-03 22:31:15 IST","user":"3paradm"},{"id":7004,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 23:11:35 IST","finishTime":"2019-06-03 23:12:49 IST","user":"3paradm"},{"id":7005,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 23:12:59 IST","finishTime":"2019-06-03 23:13:34 IST","user":"3paradm"},{"id":7006,"type":15,"name":"checkiperfserver","status":1,"startTime":"2019-06-03 23:15:00 IST","finishTime":"2019-06-03 23:15:00 IST","user":"3parsvc"},{"id":7007,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 23:17:16 IST","finishTime":"2019-06-03 23:18:01 IST","user":"3paradm"},{"id":7008,"type":20,"name":"createfpg_task","status":1,"startTime":"2019-06-03 23:26:18 IST","finishTime":"2019-06-03 23:27:32 IST","user":"3paradm"},{"id":7009,"type":15,"name":"remove_expired_vvs","status":1,"startTime":"2019-06-03 23:27:00 IST","finishTime":"2019-06-03 23:27:00 IST","user":"3parsvc"},{"id":7010,"type":20,"name":"createvfs_task","status":1,"startTime":"2019-06-03 23:27:42 IST","finishTime":"2019-06-03 23:28:16 IST","user":"3paradm"},{"id":7011,"type":20,"name":"removefpg_task","status":1,"startTime":"2019-06-03 23:31:26 IST","finishTime":"2019-06-03 23:32:11 IST","user":"3paradm"}]}